instruments/technical_docs_for_high_risk_ai.yaml

```
@@ -0,0 +1,378 @@
+systemcard_path: .assessments[]
+schema_version: 1.1.0
+
+instrument:
+  name: "Technische documentatie voor hoog-risico AI-systemen"
+  description: "De technische documentatie van een AI-systeem met een hoog risico wordt opgesteld voordat dit systeem in de handel wordt gebracht of in gebruik wordt gesteld."
+  urn: "urn:nl:aivt:ir:td:1.0"
+  language: "nl"
+  owners:
+    - organization: "European Union"
+    - email: ""
+    - role: ""
+    - date: ""
+    - url: "https://www.europarl.europa.eu/doceo/document/TA-9-2024-0138-FNL-COR01_NL.pdf" #See article 11 and annex IV of European AI Act
+
+template: &systemcard_template                  # anchor for all fields belonging to answer
```

Maybe change systemcard to assessment card, because essentially this is the template for the assessment card
```suggestion
template: &assessmentcard_template                  # anchor for all fields belonging to answer
```